src/main/resources/de/tracetronic/jenkins/plugins/ecutestexecution/steps/ProvideGeneratedReportsStep/config.properties

```diff
@@ -0,0 +1,4 @@
+step.description=This step provides all custom generated ecu.test reports as job artifacts.
+step.compatNote=This feature is only available for ecu.test 2024.3 or higher!
+selectedReportTypes.title=Select Generated Reports
+selectedReportTypes.description=Generated report name that should be provided.
```

Comment:
Add that it needs to be a comma separated list with the possibility to use wildcards